BREAKING.md

```diff
@@ -57,6 +60,39 @@ getFileLink(
 )
 ```
 
+### AgentScheduler removed as a default member of ContainerRuntime
```

Comment:
@leeviana - can you please take a look (loop right people to assess impact on next bump)? Thanks!